torch/quantization/fake_quantize.py

```diff
@@ -204,7 +204,7 @@ class FixedQParamsFakeQuantize(FakeQuantizeBase):
     def __init__(self,
                  scale,
                  zero_point,
-                 dtype,
+                 dtype=torch.quint8,
```

Comment:
note: adding this so the default for dtype aligns with the defaults for quant_min/quant_max